scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -58,6 +58,7 @@ all mutually exclusive.}
 @defidform[natbib]
 @defidform[anonymous]
 @defidform[authorversion]
+@defidform[noacm]
 @defidform[9pt]
 @defidform[10pt]
 @defidform[11pt]
```

Comment:
There's also a `@tabular` of defaults below, copied from Table 2 (page 8) of the acmart pdf. `nonacm` should be in the tabluar.
https://www.acm.org/binaries/content/assets/publications/consolidated-tex-template/acmart.pdf